pynumaflow/function/multiproc_server.py

```diff
@@ -135,7 +135,9 @@ def MapFn(
             )
         except Exception as err:
             _LOGGER.critical("UDFError, re-raising the error: %r", err, exc_info=True)
-            raise err
+            context.set_code(grpc.StatusCode.UNKNOWN)
```

Comment:
Same as here, as this is a `UDFError`.

Reply:
same here - https://grpc.github.io/grpc/core/md_doc_statuscodes.html